### PR TITLE
Fix small utils bug.

### PR DESCRIPTION
**Pull request by kenzieschmoll**
I have added some utils methods recently without tests. Added tests for those methods while I was here.

### CODE REVIEW

File: packages/devtools/lib/src/utils.dart [hunk 1]
```diff
@@ -57,13 +57,23 @@ String printMb(num bytes, [int fractionDigits = 1]) {
...
+  return msAsText(
+    micros / 1000,
+    includeUnit: includeUnit,
+    fractionDigits: fractionDigits,
+  );
 }
 
-String msAsText(num milliseconds,
-    {bool includeUnit = true, int fractionDigits = 1}) {
+String msAsText(
```
Comment (jacob314) [inline packages/devtools/lib/src/utils.dart]: Maybe change these to one method
formatTimeInMilliseconds({
  num microseconds,
  num milliseconds, ...
}) {
  assert(microseconds == null || milliseconds == null);
  if (millliseconds == null) {
     assert(microseconds != null)
     milliseconds = micros / 100;
  }
  Alternately you could use the Duration class to represent times and pass in Duration objects which would also make this a bit cleaner to compute.

File: packages/devtools/test/utils_test.dart [hunk 2]
```diff
@@ -20,5 +20,41 @@ void main() {
...
+      expect(nullSafeMax(null, null), equals(null));
+    });
+
+    test('log2', () {
+      expect(log2(1), equals(0));
+      expect(log2(1.5), equals(0));
+      expect(log2(2), equals(1));
+      expect(log2(3), equals(1));
+      expect(log2(4), equals(2));
+    });
```
Comment (jacob314) [inline packages/devtools/test/utils_test.dart]: thanks for adding tests for all this. Having good unit test and integration tests will only become more important as we get more users.

File: packages/devtools/lib/src/utils.dart (hunk 1 above)
Comment (kenzieschmoll) [reply packages/devtools/lib/src/utils.dart]: done.

File: packages/devtools/lib/src/timeline/frame_flame_chart.dart [hunk 3]
```diff
@@ -466,8 +467,8 @@ class TimelineGrid extends CoreElement {
       // TODO(kenzie): Instead of calculating timestamp based on position, track
       // timestamp var and increment it by time interval represented by each
       // grid item. See comment on https://github.com/flutter/devtools/pull/325.
-      final timestamp = getTimestampForPosition(left + interval);
-      final gridItem = TimelineGridItem(left, interval, timestamp);
+      final timestampMs = getTimestampForPosition(left + interval);
```
Comment (jacob314) [inline packages/devtools/lib/src/timeline/frame_flame_chart.dart]: this all looks good. Maybe convert to Duration objects sooner? The fact that the variable names are better indicating a unit suggests we might want to store them in an object containing a unit.

File: packages/devtools/lib/src/timeline/frame_flame_chart.dart [hunk 4]
```diff
@@ -524,15 +525,15 @@ class TimelineGrid extends CoreElement {
...
+  TimelineGridItem(this.currentLeft, this.currentWidth, this.timestampMs)
       : super('div', classes: 'flame-chart-grid-item') {
     _initGridItem();
   }
 
   static const gridLineWidth = 1;
   static const timestampPadding = 4;
 
-  final num timestamp;
+  final num timestampMs;
```
Comment (jacob314) [inline packages/devtools/lib/src/timeline/frame_flame_chart.dart]: nit: change to a Duration object?

Comment (kenzieschmoll) [reply packages/devtools/lib/src/timeline/frame_flame_chart.dart]: done.

File: packages/devtools/lib/src/timeline/frame_flame_chart.dart (hunk 3 above)
Comment (kenzieschmoll) [reply packages/devtools/lib/src/timeline/frame_flame_chart.dart]: done.

**Pull request merged**